### PR TITLE
V5

### PR DESCRIPTION
**Pull request by insberr**
v2 was overhyped and the code was a mess. Here is a more neat complete rewrite.

### CODE REVIEW

**Comment by insberr** [comment]
Organize code and fix typescript errors and warnings.
Then we should prepare for merge with master

**Comment by webmsgr** [comment]
"should fix maskable icons"
it didnt

**Comment by insberr** [comment]
Are the icons working at all, at least on apple?

**Comment by insberr** [comment]
Think it's ready for review and merge with master!

**Review by insberr — COMMENTED** [review]
Looks good, review and merge. Soon ™️ . V5 moment

File: src/config/events.ts [hunk 1]
```diff
@@ -199,6 +228,29 @@ export const scheduleEvents: ScheduleEvents = [
             },
         },
     },
+    {
+        schedule: null,
+        info: {
+            message: "Marry Christmas!",
```
Comment (webmsgr) [inline src/config/events.ts:234]: marry christmas? i dont even know them!

**Review by webmsgr — DISMISSED** [review]
please let me know if the marry christmas thing was intentional, its merry

**Review by insberr — COMMENTED** [review]
reviews

File: src/config/events.ts (hunk 1 above)
Comment (insberr) [reply src/config/events.ts]: fixed. dunno in which commit tho.

**Comment by insberr** [comment]
Merge to master when

**Comment by webmsgr** [comment]
Next TODO: fix the service worker

**Comment by insberr** [comment]
Ready for your review @webmsgr. Lets merge to master soon ™️

**Pull request merged**